### PR TITLE
#144 The current implementation of MWE metric expects information

### PR DESCRIPTION
**Pull request by uurl**
For #144

**Issue**
The current implementation of MWE metric expects information about LDA topics.

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#241](https://github.com/yegor256/jpeek/issues/241) is now in scope, role is `REV`

**Comment by uurl** [comment]
@rultor merge

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` are banned at [#241](https://github.com/yegor256/jpeek/issues/241); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: src/main/resources/org/jpeek/metrics/MWE.xsl [hunk 1]
```diff
@@ -44,12 +44,14 @@ SOFTWARE.
...
-     Ensure that the core part of JPeek provides the information about topics and their probability
-     and update the metric in accordance with the provided XML structure.
+    @todo #17:30min To calculate the Maximal Weighted Entropy(MWE)
+     the "topic" is needed. To calculate the topics: a method contains words
+     excluding programming language keywords, stop words and punctuation.
+     So, to calculate the MWE metric each variable's name is a topic.
+     A High Cohesive Class has two characteristics:
+     - High Occupancy (Oi): the main topic is present in all the methods.
+     - High Distribution (Di): all the methods are related to the same topic.
+     For more information check the PDF [lyu09] in the papers.
```
Comment (yegor256) [inline src/main/resources/org/jpeek/metrics/MWE.xsl]: @uurl it's not clear from this text what exactly do you want a programmer to do. What is the task?

**Comment by yegor256** [comment]
@uurl see above

**Comment by 0crat** [comment]
The job [#241](https://github.com/yegor256/jpeek/issues/241) is now out of scope

**Pull request closed without merging**